cpp/roaring.hh

```diff
@@ -166,6 +183,14 @@ public:
         api::roaring_bitmap_add_many(&roaring, n_args, vals);
     }
 
+    /**
+     * Add value val, using context from a previous insert for speed
+     * optimization.
+     */
+    void addBulk(BulkContextWrapper *wrapper, uint32_t x) noexcept {
```

Comment:
Could we pass a reference to wrapper instead of a pointer?

Reply:
Sure, I've done it.